tensorboard/plugins/debugger_v2/debugger_v2_plugin_test.py

```diff
@@ -27,9 +29,14 @@
 from tensorboard.backend import application
 from tensorboard.plugins import base_plugin
 from tensorboard.plugins.debugger_v2 import debugger_v2_plugin
+from tensorboard.plugins.debugger_v2 import debug_data_multiplexer
```

Comment:
nit: alphabetize import (`debug_*` < `debugger_*`)

Reply:
No, the TensorBoard CI never complained about out-of-order imports (at
least, not any time in the past year or so). Our `flake8` checks are
minimal, only looking for obvious problems like syntax errors or unused
or undefined variables. Thanks for fixing. :-)